ts.js

```diff
@@ -124,7 +125,28 @@
          }
       ],
       '@typescript-eslint/no-inferrable-types': ['error', {'ignoreParameters': true}],
-      '@typescript-eslint/no-misused-promises': ['error', {'checksConditionals': true, 'checksVoidReturn': false}]
+      '@typescript-eslint/no-misused-promises': ['error', {'checksConditionals': true, 'checksVoidReturn': false}],
+      '@typescript-eslint/class-literal-property-style': 'error',
+      '@typescript-eslint/no-confusing-non-null-assertion': 'error',
+      '@typescript-eslint/no-extraneous-class': 'error',
+      '@typescript-eslint/no-meaningless-void-operator': 'error',
+      '@typescript-eslint/no-non-null-asserted-nullish-coalescing': 'error',
+      'no-throw-literal': 'off',
+      '@typescript-eslint/no-throw-literal': 'error',
+      '@typescript-eslint/no-unnecessary-boolean-literal-compare': 'error',
+      '@typescript-eslint/no-unnecessary-condition': 'error',
+      'no-useless-constructor': 'off',
+      '@typescript-eslint/no-useless-constructor': 'error',
+      '@typescript-eslint/non-nullable-type-assertion-style': 'error',
+      '@typescript-eslint/prefer-for-of': 'error',
+      '@typescript-eslint/prefer-function-type': 'error',
```

Comment:
this one confuses me by name, but from the docs it appears to encourage you to use function declares like `(): thing` as opposed to `Function` and has nothing to do with using `function` vs `var thing = () =>`, so I think it's good?  by name it sounds like it wants you to use `function XXX` declarations or something.

Reply:
yea it's just a minor style thing in defining types. I don't think I even ran into it in Audiate when I brought this in.